source/adapters/opencl/program.cpp

```diff
@@ -220,6 +220,26 @@ urProgramLink(ur_context_handle_t hContext, uint32_t count,
   return UR_RESULT_SUCCESS;
 }
 
+UR_APIEXPORT ur_result_t UR_APICALL urProgramCompileExp(ur_program_handle_t,
```

Comment:
These should also be added to `ur_interface_loader.cpp`

Reply:
Done.